instrumentation/servlet/servlet-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v5_0/dispatcher/RequestDispatcherAdvice.java

```diff
@@ -39,21 +37,15 @@ public static void start(
 
     // this tells the dispatched servlet to use the current span as the parent for its work
     request.setAttribute(CONTEXT_ATTRIBUTE, context);
-
-    // TODO (trask) do we need this, since doing manual propagation above?
-    scope = context.makeCurrent();
```

Comment:
Interesting.. which then leads to removing this instrumentation altogether.. let's see what CI says about this 😄

Reply:
> which then leads to removing this instrumentation altogether..

That's the best outcome possible 😄

Reply:
I tried to find a use case where it would be needed, but failed. I believe `Servlet3Advice` `needsRescoping` already handles the same thing as `RequestDispatcherAdvice`. Burn it with fire :fire:

Reply:
💯 if we find a need for it later, that's fine, but until then 🔥 it is